<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommWidget</class>
 <widget class="QWidget" name="CommWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>374</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1068</width>
    <height>374</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1068</width>
    <height>374</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="comm_scroll">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>1061</width>
     <height>331</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1061</width>
     <height>331</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1061</width>
     <height>331</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QWidget" name="comm_scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1059</width>
      <height>329</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="comm_layout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="comm_addUnit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Unit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="comm_remUnit">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>10</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Unit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
